test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -267,82 +270,470 @@ var _ = g.Describe("Scylla Manager integration", func() {
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Verifying that backup task deletion was synchronized")
+		tasks, err = managerClient.ListTasks(ctx, *sourceSC.Status.ManagerID, "backup", false, "", "")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(tasks.TaskListItemSlice).To(o.BeEmpty())
+
+		// Close the existing session to avoid polluting the logs.
+		di.Close()
+
+		framework.By("Deleting source ScyllaCluster")
+		var propagationPolicy = metav1.DeletePropagationForeground
+		err = f.ScyllaClient().ScyllaV1().ScyllaClusters(sourceSC.Namespace).Delete(ctx, sourceSC.Name, metav1.DeleteOptions{
+			Preconditions: &metav1.Preconditions{
+				UID: &sourceSC.UID,
+			},
+			PropagationPolicy: &propagationPolicy,
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		waitCtx7, waitCtx7Cancel := context.WithCancel(ctx)
+		defer waitCtx7Cancel()
+		err = framework.WaitForObjectDeletion(
+			waitCtx7,
+			f.DynamicClient(),
+			scyllav1.GroupVersion.WithResource("scyllaclusters"),
+			sourceSC.Namespace,
+			sourceSC.Name,
+			&sourceSC.UID,
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		targetSC := f.GetDefaultScyllaCluster()
+		targetSC.Spec.Datacenter.Racks[0].Members = sourceSC.Spec.Datacenter.Racks[0].Members
+		targetSC.Spec.ScyllaArgs = "--consistent-cluster-management=false"
+
+		switch objectStorageType {
+		case framework.ObjectStorageTypeGCS:
+			targetSC = provideGCSCredentials(ctx, targetSC)
+		default:
+			g.Fail("unsupported object storage type")
+		}
+
+		framework.By("Creating target ScyllaCluster")
+		targetSC, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, targetSC, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for target ScyllaCluster to roll out (RV=%s)", targetSC.ResourceVersion)
+		waitCtx8, waitCtx8Cancel := utils.ContextForRollout(ctx, targetSC)
+		defer waitCtx8Cancel()
+		targetSC, err = controllerhelpers.WaitForScyllaClusterState(waitCtx8, f.ScyllaClient().ScyllaV1().ScyllaClusters(targetSC.Namespace), targetSC.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), targetSC)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), targetSC)
+
+		framework.By("Verifying that target cluster is missing the source cluster data")
+		targetHosts, err := utils.GetBroadcastRPCAddresses(ctx, f.KubeClient().CoreV1(), targetSC)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(targetHosts).To(o.HaveLen(int(utils.GetMemberCount(targetSC))))
+		err = di.SetClientEndpoints(targetHosts)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		err = di.AwaitSchemaAgreement(ctx)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		_, err = di.Read()
+		o.Expect(err).To(o.HaveOccurred())
```

Comment:
Can we somehow check that this is error about missing data and not about anything else?

Reply:
I'll see if we can compare the errors or if we'd have to parse the message

Reply:
not perfect, but done